HtmlRenderer/Renderer.cs

```diff
@@ -185,8 +186,8 @@ protected virtual WindowInfo CreateWindowInfo()
 
         public void BeginRender()
         {
-            EndRender();
 
+            EndRender();
```

Comment:
`EndRender()` here sets `isStopRender` to true, what if event comes in before it setted to false?

I think we should use something more solid.